interfaces/svg-animations.idl

```diff
@@ -0,0 +1,60 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content was automatically extracted by Reffy into reffy-reports
+// (https://github.com/tidoust/reffy-reports)
+// Source: SVG Animations (https://svgwg.org/specs/animations/)
+
+interface TimeEvent : Event {
+
+  readonly attribute @@ unknown term "AbstractView" view;
```

Comment:
'@@ unknown term "AbstractView"'

^ well thats unusual. It's actually in the IDL at https://svgwg.org/specs/animations/#InterfaceTimeEvent (scroll down a bit from the anchor), so I think we need to patch the spec

Reply:
And turns out foolip already did. https://github.com/w3c/svgwg/issues/775

Reply:
The blocking SVG issue has been fixed, yay!